projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/ai-image/featured-image.tsx

```diff
@@ -51,28 +51,28 @@ export default function FeaturedImage( {
 	);
 	const siteType = useSiteType();
 	const postContent = usePostContent();
-	const { postTitle, postFeaturedMedia, isEditorPanelOpened } = useSelect(
+	const { postTitle, postFeaturedMediaURL, postFeaturedMediaId, isEditorPanelOpened } = useSelect(
 		( select: ( store ) => EditorSelectors ) => {
-			const featuredMediaId = select( editorStore ).getEditedPostAttribute( 'featured_media' );
+			const editorStoreSelect = select( editorStore );
+			const featuredMediaURL = editorStoreSelect.getEditedPostAttribute(
+				'jetpack_featured_media_url'
```

Comment:
Using `jetpack_featured_media_url` to decide is a bit more of a nuance (I think). The post attribute is only set on saving, not immediate

Reply:
`jetpack_featured_media_url` is added on save as a backend API field hook. It should always respond to the current media ID (`featured_media` attribute). If we add this as a param to rely on I think we'll be introducing yet another point where things can go out of sync or be in a "dirty" state. I'd stick to just considering `featured_media` attribute.